src/components/Progressbar.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC, PropsWithChildren } from 'react';
+
+export type ProgressbarProps = PropsWithChildren<{
+  progress?: string | '10%' | '20%' | '30%' | '40%' | '50%' | '60%' | '70%' | '80%' | '90%' | '100%';
```

Comment:
@k2glyph I know that it still WIP but, guess that would be better to just use number instead string?

Reply:
okay thanks @rluders. Sure